velocity_planner/velocity_planner/add_imu_frame.py

```diff
@@ -1,3 +1,4 @@
+# What is the IMU frame for?
```

Comment:
Looking at this file, it doesn't do anything. I'll delete it. I was just playing around with the rosbags API and added it to git accidentally I think. In general, the IMU frame is the coordinate frame of reference of the IMU sensor.